modules/integration_azure-datafactory/conf/02-adf-pipeline-run.yaml

```diff
@@ -0,0 +1,24 @@
+module: "Azure DataFactory"
+name: "pipeline error rate"
+filtering: "filter('resource_type', 'Microsoft.DataFactory/factories') and filter('primary_aggregation_type', 'true')"
+aggregation: ".sum(by=['name', 'azure_resource_group_name', 'azure_region'])"
+transformation: ".min(over='5m')"
+value_unit: "%"
+signals:
+  adf_pipeline_succeeded_run:
+    metric: "PipelineSucceededRuns"
+    rollup: rate
```

Comment:
while you make a percentage the default rollum sum will give the same result and it seems to be a most relevant value for troubleshooting